.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,37 +1,6 @@
+#### Summary
 <!--
-SPDX-FileCopyrightText: 2017-2020 City of Espoo
```

Comment:
Could you add the whole `.github` directory to the bulk licensing file (example: https://github.com/espoon-voltti/evaka/blob/master/.reuse/dep5#L17-L20)

Reply:
Oh and I'm just too late